app/src/main/java/org/ole/planet/myplanet/ui/team/MyTeamsDetailFragment.kt

```diff
@@ -112,8 +115,10 @@ class MyTeamsDetailFragment : BaseNewsFragment() {
                 map["message"] = msg
                 map["messageType"] = team?.teamType!!
                 map["messagePlanetCode"] = team?.teamPlanetCode!!
-                createNews(map, mRealm, user, imageList)
-                rvDiscussion.adapter?.notifyDataSetChanged()
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
1. devices that are not within this range will not be able to create news based on the condition you created